examples/mongo/README.md

```diff
@@ -0,0 +1,25 @@
+# Mongo Metrics
+
+The [`mongo_3.rb`](mongo_3.rb) script generates metrics from the `mongotop` and `mongostat` commands for a Mongo 3.0 database. The following metrics will be output for every collection in your system:
```

Comment:
Mongo 2 users... upgrade mongo if you want data?

A lot of this looks like it might work for mongo 2, not sure what demand is like for that.